terraform/infrastructure/samples/aws/sns.tf

```diff
@@ -0,0 +1,12 @@
+# AWS S3 bucket with versioning
+# For more details please see official provider documentation https://registry.terraform.io/modules/terraform-aws-modules/sns/aws/latest
```

Comment:
suggestion to change this resource 
https://registry.terraform.io/modules/terraform-aws-modules/sqs/aws/latest#subscribe-queue-to-sns-topic

Reply:
Changed!